documentation/src/main/asciidoc/user_guide/eviction.adoc

```diff
@@ -87,19 +109,38 @@ Programmatically, the same would be defined using:
 [source,java]
 ----
 Configuration c = new ConfigurationBuilder().eviction().strategy(EvictionStrategy.LRU)
-               .maxEntries(2000).expiration().wakeUpInterval(5000l).lifespan(1000l).maxIdle(500l)
+               .size(2000).expiration().wakeUpInterval(5000l).lifespan(1000l).maxIdle(500l)
                .build();
 
 ----
 
+==== Memory Based Eviction Configuration
+Memory based eviction may require some additional configuration options if you
+are using your own custom types as Infinispan is normally used.  In this case
+Infinispan cannot estimate the memory usage of your classes and as such you are
+required to use `storeAsBinary` when memory based eviction is used.
```

Comment:
* Start sentence on new line.
* Parenthesis around (as Infinispan is normally used)?
* "you are required to use `storeAsBinary" is a bit vague, do we log a warning or throw an exception when an application inserts a custom type in a cache with memory based eviction, or is it just a risk of OOME?
* I'd change the examples to include `storeAsBinary` as well.